Alcolator/AppDelegate.m

```diff
@@ -17,17 +18,39 @@ @interface AppDelegate ()
 @implementation AppDelegate
 
 
+
+
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]] ;
     // Override point for customization after application launch.
-    ViewController *viewController = [[ViewController alloc] init];
-    self.window.rootViewController = viewController;
+    ViewController *wineVC = [[ViewController alloc] init];
+    WhiskeyViewController *whiskeyVC = [[WhiskeyViewController alloc] init];
+    UITabBarController *tabBarVC = [[UITabBarController alloc] init];
+    tabBarVC.viewControllers = @[wineVC, whiskeyVC];
+    
+    tabBarVC.delegate = self; // don't understand this
```

Comment:
This line is selling the UITabBarController that you implement the UITabBarDelegate protocol and want to be notified about certain events (by implementing one or more of the UITabBarDelegate methods). In this case you are implementing the tabBarController:didSelectViewController: method to get notified when someone changes the tab.